source/src_pw/potential_libxc_meta.cpp

```diff
@@ -227,7 +227,8 @@ tuple<double,double,ModuleBase::matrix,ModuleBase::matrix> Potential_Libxc::v_xc
 
 			if (rh > rho_th && ggrho2 > grho_th && abs(atau) > tau_th)
 			{
-				XC_Functional::tau_xc_spin(rhoup, rhodw, grhoup, grhodw, tauup, taudw, ex, ec, v1xup, v1xdw, v2xup, v2xdw, v3xup, v3xdw, v1cup, v1cdw, v2cup, v2cdw, v3cup, v3cdw );
+				vector<double> v2c;
+				XC_Functional::tau_xc_spin(rhoup, rhodw, grhoup, grhodw, tauup, taudw, ex, ec, v1xup, v1xdw, v2xup, v2xdw, v3xup, v3xdw, v1cup, v1cdw, v2cup, v2cdw, v2c, v3cup, v3cdw );
```

Comment:
please don't write too many parameters in one line, if you want your code to be convenient to read, you can use a new function to replace this, just like:
"
void my_tau_xc_spin(std::array<double, 21> xc_inputs)
{...}
"